src/sdk.js

```diff
@@ -0,0 +1,70 @@
+import Web3 from 'web3';
+import IPFSClient from 'ipfs-http-client';
+import url from 'url';
+import RegistryNetworks from 'singularitynet-platform-contracts/networks/Registry.json';
+import RegistryAbi from 'singularitynet-platform-contracts/abi/Registry.json';
+
+import Account from './Account';
+import MPEContract from './MPEContract';
+import ServiceClient from './ServiceClient';
+import { find } from 'lodash';
+
+const DEFAULT_CONFIG = {
+  defaultGasLimit: 210000,
+  defaultGasPrice: 4700000,
+  ipfsEndpoint: 'http://ipfs.singularitynet.io:80',
+};
+
+export default class SnetSDK {
+  constructor(config) {
+    this._config = {
+      ...DEFAULT_CONFIG,
+      ...config,
+    };
+    const options = {
+      defaultGas: this._config.defaultGasLimit,
+      defaultGasPrice: this._config.defaultGasPrice,
+    };
+    this._networkId = config.networkId;
+    this._web3 = new Web3(config.web3Provider, null, options);
+    this._mpeContract = new MPEContract(this._web3, this._networkId);
+    this._account = new Account(this._web3, this._networkId, this._config, this._mpeContract);
+    this._web3.eth.defaultAccount = this._account.address;
+    const registryAddress = RegistryNetworks[this._networkId].address;
+    this._registryContract = new this._web3.eth.Contract(RegistryAbi, registryAddress, { from: this._account.address });
+  }
+
+  get account() {
+    return this._account;
+  }
+
+  get web3() {
+    return this._web3;
+  }
+
+  async createServiceClient(orgId, serviceId, groupName, paymentChannelManagementStrategy, ServiceStub, options = {}) {
```

Comment:
Sorry for a late question, I have already asked it in sample review [here](https://github.com/singnet/snet-code-examples/pull/2#discussion_r285569162). Just duplicating it here:
1. Is it really necessary to pass `service.CalculatorClient` as last parameter? We could export all services as part of `stub` callection of the `serviceClient`.
2. As far as I remember @vforvalerio87 suggested using `service` instead of `stub` and adding `serviceClient.messages` to return messages classes

Reply:
1. So given a grpc protobuf we will not be able to determine which stub to use if there are more than one stubs available in the protobuf. 
2. Since this version is behaving like a statically typed sdk, we will have to accept messages as well if we want to be able to expose something like `serviceClient.messages`. We could rename `stub` to `service`, since the instance on which we would call `service` itself was `serviceClient` I thought of naming it as `stub`. Let me know if I should change `stub` to `service`.

Reply:
We will handle this in the dynamic version of the SDK.